docs.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -e
+
+# Setup:
+#  mix escript.install hex ex_doc
+
+# Usage:
+# ./docs.sh <source-ref>
+
+SOURCE_REF=$1
+
+version=1.2.0-dev
+
+rebar3 compile
+rebar3 as docs edoc
```

Comment:
so it builds both `edoc` and `ExDoc`? Or is it just to produce documentation chunks?
Because what I was actually about to propose is that we test that we still able to build both Erlang edocs and elixir ExDoc, even if we are still going to publish only one of them.

Reply:
Sadly not. The change in `rebar.confic` instructs `edoc` to generate doc chunks instead of HTML.
I would also prefer to build both as a starting point.

I've not checked if `rebar3 edoc` accepts this options somehow as command line parameters?

Reply:
Ah, ok. Well, I don't have any strong opinion on this topic.
I guess it should be possible to re-define the `edoc_opts` by creating some additional profile in `rebar.config`, but I'm not sure if it worth it.

> I've not checked if rebar3 edoc accepts this options somehow as command line parameters?

It seems `edoc` command doesn't take any parameters: https://github.com/erlang/rebar3/blob/ba50a0828eba37d386c5f82cf1814d4519d66f64/src/rebar_prv_edoc.erl#L28

```
$ rebar3 help edoc
===> Analyzing applications...
Generate documentation using edoc.
Usage: rebar3 edoc
```

Reply:
Fine. So let us then keep this consideration at side for now.
Locally it can be anyway done be commenting out the `edoc_opts` in the `rebar.config` and build doc wit EDoc as usual.